extensions/graphviz_support/graphviz_support/tests/test_lg_graphviz_api.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+import unittest
+import os
+import pathlib
+import labgraph as lg
+from labgraph.examples.simple_viz import Demo
+from ..errors.errors import GenerateGraphvizError
+from ..generate_graphviz.generate_graphviz import (
+    identify_graph_nodes,
+    find_connections,
+    generate_graphviz
+)
+
+
+class TestLabgraphGraphvizAPI(unittest.TestCase):
+
+    def setUp(self) -> None:
```

Comment:
setup

Reply:
setUp() is a method of the  unittest.TestCase class
ref : https://docs.python.org/3/library/unittest.html#classes-and-functions